scheduler-server/scheduling-dsl-compiler/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "typescript-compiler",
+  "version": "1.0.0",
+  "description": "Typescript compiler for the scheduling server DSL",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "test": "npm run build && node build/main.js",
+    "generate-ast-schema": "npx typescript-json-schema src/libs/scheduler-ast.ts \"*\" --aliasRefs --out ./build/scheduler.schema.json"
+  },
+  "author": "",
+  "dependencies": {
+    "commander": "^9.0.0",
+    "piscina": "^3.2.0",
```

Comment:
Not needed anymore

Reply:
Good catch

Reply:
commander as well i think

Reply:
Removed.

Reply:
I don't even think we're using source-map yet?

Reply:
Latest force push removed commander.